addon/adapters/rest.js

```diff
@@ -1094,6 +1094,27 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
     $.ajax(options);
   },
 
+  /**
+    @method _najaxRequest
+    @private
+    @param {Object} options jQuery ajax options to be used for the najax request
+  */
+  _najaxRequest(options) {
+    if (najax) {
```

Comment:
I think this needs to be `typeof najax !== ‘undefined’`

Reply:
done